lib/aweplug/helpers/video/video_base.rb

```diff
@@ -39,32 +39,30 @@ def initialize(video, site, default_ttl = 86400) # A day seems good for videos
           end
         end
 
-        def description(full_description=false)
-          max_length=150
+        def description
           d = @video["description"]
           out = ""
           if d
-            if full_description
-              max_length = d.length
-            end
             i = 0
+            max_length = 150
             d.scan(/[^\.!?]+[\.!?]\s/).map(&:strip).each do |s|
               i += s.length
+              out += s
+
               if i > max_length
-                out = s[0..max_length]
                 break
-              else
-                out += s
               end
             end
             # Deal with the case that the description has no sentence end in it
             out = (out.empty? || out.length < 60) ? d : out
           end
-          out = out.gsub("\n", ' ')[0..max_length]
+          out = out.gsub("\n", ' ')[0..150]
```

Comment:
This should still be max_length, and line 47 should be move up outside of that if block